src/main/scala/tech/christopherdavenport/twitterstorm/Server.scala

```diff
@@ -117,21 +117,30 @@ case class Server[F[_]](tweets: Stream[F, BasicTweet])(
         }
     }
 
+  import Syntax._
+
   def serve(port: Int, ip: String, topN: Int, maxQueueSize: Int, emojiResource: String): Stream[F, Nothing] = {
     val logHeaders = true
     val logBody = true
-    for {
-      scheduler <- Scheduler[F](3)
-      timer <- fs2.async.hold(Duration.Zero, scheduler.awakeEvery(10.millis))
-      counter <- Stream.eval(fs2.async.signalOf[F, BigInt](0))
-      emojiMap <- EmojiParser.emojiMapFromResource(emojiResource)
-      reporter <- tweets.through(StreamTweetReporter(emojiMap, topN, maxQueueSize))
-      nothing <- BlazeBuilder[F] // Stream[F, Nothing] I Would love to remove deadCode Warning Here.
+    (for {
+      scheduler <- Resource(Scheduler[F](3))
+      timer <- Resource(fs2.async.hold(Duration.Zero, scheduler.awakeEvery(10.millis)))
+      counter <- Resource.eval(fs2.async.signalOf[F, BigInt](0))
+      emojiMap <- Resource(EmojiParser.emojiMapFromResource(emojiResource))
+      reporter <- Resource(tweets.through(StreamTweetReporter(emojiMap, topN, maxQueueSize)))
+      server <- BlazeBuilder[F] // Stream[F, Nothing] I Would love to remove deadCode Warning Here.
         .bindHttp(port, ip)
         .mountService(service(counter, timer), "/util")
         .mountService(Logger(logHeaders, logBody)(twitterService(reporter)), "/twitter") // Logger for Console Visualization
-        .serve
-    } yield nothing
+        .resource
+    } yield server).serve
+  }
+}
+
+object Syntax {
+  implicit class ServerBuilderSyntax[F[_]](val sb: org.http4s.server.ServerBuilder[F]) extends AnyVal {
```

Comment:
This would be provided by http4s if http4s provided the `Resource`, though it's more code that isn't about HTTP.